docs/topics/braintree.rst

```diff
@@ -131,12 +131,23 @@ solitude.
             }
         }
 
-    :>json string braintree token: id of the payment method in braintree.
+    :>json string braintree token: id of the payment method in Braintree.
     :>json string braintree created_at: created date and time.
     :>json string braintree updated_at: updated date and time.
 
     :status 201: payment method created.
 
+To cancel a payment method. This will delete the payment method in Braintree
+and is not reversible.
+
+.. http:post:: /braintree/paymethod/cancel/
+
+    :<json string paymethod: the resource_uri of the payment method in solitude.
+
+    The response is in the same format as for creation.
```

Comment:
This seems weird to me because the response has created_at timestamps https://solitude.readthedocs.org/en/latest/topics/braintree.html#post--braintree-paymethod-